api/src/test/java/org/hawkular/apm/api/utils/EndpointUtilTest.java

```diff
@@ -96,6 +96,8 @@ public void testDecodeNonClientURI() {
     @Test
     public void testGetSourceEndpointConsumer() {
         Trace trace = new Trace();
+        trace.setTraceId("1");
```

Comment:
The decision regarding whether to add the 'client:' prefix has to test whether the trace fragment is the initial (root) fragment, which is achieved by checking that the fragment and trace ids are the same. This required these tests to be updated to set the trace id, which was previously not required.